docs/testplans/study-viewer/study-viewer.md

```diff
@@ -0,0 +1,30 @@
+# Study Viewer
+
+## Overview
+Dataset Browser allows the user visiting the commons to discover if the dataset they need is available on the commons. The user doesn't need to have full access to look for the dataset they are interested in.
+
+It enables to the user to check :
+1. if Gen3 has the data they need
+2. if the data is sufficient enough for research 
+
+## Technical Document
+[Dataset browser/ Study Viewer](https://docs.google.com/document/d/1BLbLX4GEViJfcWFNDNN723KmC2XMHH-SWFMM_cPg6wk/)
+
+[Access Request Queue](https://docs.google.com/document/d/1h5ZLYXb_wi2a2H3sfXrRcY41KQ0SjQF3DBtdpt2oLxE/)
+
+## Workflow
+
+1. The users visits the commons 
+2. Navigate to the Dataset Browser/ Study Viewer Tab on the navigation bar
+3. The users searches for the dataset that they need and anc clicks 'Show Details'
+4. if they have access, 'Download' button is displayed
+5. if they do not have access, 'Request Access' button is displayed
+6. Click on download button, if file -> file is downloaded, if program/project/study -> JSON manifest is downloaded
```

Comment:
the first iteration will only allow object file download, no JSON manifest, so i think that can be omitted for now. we're not sure yet it will be part of the final feature at all, since we'd like to use PFB instead